.github/workflows/mvn_package_deploy.yml

```
@@ -98,4 +98,4 @@ jobs:
                 run: mvn -f ${{ inputs.parent_pom }} versions:commit
 
             -   name: Deploy package
-                run: mvn -f ${{ inputs.parent_pom }} deploy -DskipTests
+                run: mvn -f ${{ inputs.parent_pom }} deploy -DskipTests -DaltDeploymentRepository="ubique-artifactory::${{ secrets.artifactory_url }}${{ secrets.artifactory_repo }}"
```

```suggestion
                run: mvn -f ${{ inputs.parent_pom }} deploy -DskipTests -DaltDeploymentRepository="${{ inputs.artifactory_id }}::${{ secrets.artifactory_url }}${{ secrets.artifactory_repo }}"
```

Pass a new input `artifactory_id` and use it in the above action above. Set the default of this input to `ubique-artifactory` to make it backward-compatible.

---

As far as I understand, this server/repo id is only used to refer to the entry in the settings.xml that we generate in the same workflow in the step "Create custom artifactory settings.xml" above. The id string is thus entirely arbitrary and we could make it look more generic by using an id like "repo".
So, making this configurable would not seem useful. Am I misunderstanding something?